app/Models/Photo.php

```diff
@@ -104,8 +102,6 @@ class Photo extends Model implements HasRandomID
 		'taken_at' => DateTimeWithTimezoneCast::class,
 		'live_photo_full_path' => MustNotSetCast::class . ':live_photo_short_path',
 		'live_photo_url' => MustNotSetCast::class . ':live_photo_short_path',
-		'is_downloadable' => MustNotSetCast::class,
-		'is_share_button_visible' => MustNotSetCast::class,
```

Comment:
I know that we decided to remove `is_share_button_visible` from an per-album/per-photo level and make it a global configuration option. But what happened to `is_downloadable`?

Reply:
`is_downloadable` is `grants_download`.